src/components/TopBar/TopBar.vue

```diff
@@ -60,7 +60,7 @@
 				:aria-label="threadNotificationLabel"
 				:title="threadNotificationLabel"
 				:primary="!!threadNotification"
-				@click="() => {}">
+				@click="chatExtrasStore.setThreadNotificationLevel(token, threadId, threadNotification ? 0 : 1)">
```

Comment:
We should allow all 4 notification levels?
- Default (basically reset to conversation level)
- Always
- Mention
- Never

?

And if we say "that's too much", we should toggle between 1 and 3 (always and never), and not always and default (which could be the same, as many people asked for setting the group default also to always)

Reply:
We should, that's just a plug for now

Reply:
Yeah, but these make sense inside of the menu only, and it's missing 'default'

On the outside, we should see some variations of a bell icon

Reply:
Always / All messages could use Chat or Comment, or the Reply icon?
<img width="347" height="362" alt="grafik" src="https://github.com/user-attachments/assets/3778ea87-e784-4531-9262-92d76a7e2924" />

Reply:
default | other | never
-- | -- | --
<img width="344" height="314" alt="image" src="https://github.com/user-attachments/assets/551e9114-2a2f-491b-b951-f848c4a55715" /> | <img width="348" height="325" alt="image" src="https://github.com/user-attachments/assets/553c6904-07db-4128-a5be-c2787ef0e52f" /> | <img width="350" height="328" alt="image" src="https://github.com/user-attachments/assets/bcd7ff1f-f550-42ce-822f-06e1b711d258" />

Reply:
For `@` we can use a bell with `@` sign on the bottom right

Reply:
> If such icon would exist

We can always stack one icon on top of another 👀

Reply:
(Replying directly as @nimishavijay  is out today :)
We don’t need to have icons which all include the notification bell, as they would also be more difficult to tell apart.

Is there a specific issue about the current icons that needs to be solved? If we want to switch from the "person" icon to an "@" that’s fine too, however it repeats the first character of the text right next to it.